packages/app/components/Nav/NotificationSettingsModal.tsx

```diff
@@ -67,34 +80,17 @@ export function NotificationSettingsModal({
     >
       {profile && (
         <Form form={form} initialValues={profile}>
-          <p>Get notified when you reach the top of the queue.</p>
           <Form.Item
-            label="Enable browser notifications"
+            label="Enable notifications on all devices"
             name="desktopNotifsEnabled"
             valuePropName="checked"
           >
-            <Switch
-              onChange={async (checked) => {
-                if (checked) {
-                  const canNotify = await requestNotificationPermission();
-                  if (canNotify !== NotificationStates.granted) {
-                    form.setFieldsValue({ desktopNotifsEnabled: false });
-                    form.setFields([
-                      {
-                        name: "desktopNotifsEnabled",
-                        errors: [
-                          canNotify === NotificationStates.notAllowed
-                            ? "Please allow notifications in this browser"
-                            : "Browser does not support notifications. Please use Chrome or Firefox.",
-                        ],
-                      },
-                    ]);
-                  }
-                }
-              }}
-            />
+            <Switch />
```

Comment:
Do we still need this? Since user can now add/remove individual devices it seems like it might confuse some people if they accidentally forgot to toggle the switch on.

Reply:
yeah I'll change it to default on, and run a sql query on prod to turn current user's on